packaging/windows/logdna.conf

```diff
@@ -0,0 +1,23 @@
+http:
+  host: logs.logdna.com
+  endpoint: /logs/agent
+  use_ssl: true
+  timeout: 10000
+  use_compression: true
+  gzip_level: 2
+  body_size: 2097152
+  ingestion_key: <YOUR_INGESTION_KEY>
```

Comment:
Should we add the params attribute below the ingestion_key as part of the default template config?
 For example:
      params:
        hostname: 
        now: 0

Reply:
no. Chris fixed the default hostname bug already.